_config.yml

```diff
@@ -1,91 +1,73 @@
-title: 	NEON Raster Data in R 
-description: 	A set of lessons .
-#Comment out url when working locally to resolve base urls correctly ok
-#use this when pushing to git for live version
+#------------------------------------------------------------
+# Values for this lesson.
+#------------------------------------------------------------
 
-url:      http://datacarpentry.org/NEON-R-Spatial-Raster
-baseurl:   http://datacarpentry.org/NEON-R-Spatial-Raster
+# Which carpentry is this ("swc", "dc", or "lc")?
+carpentry: "dc"
 
-#use this when working locally
-#just uncomment the first localhost.
-#url: http://localhost:4000
-#baseurl: http://localhost:4000
+# Overall title for pages.
+title: "Geospatial analysis with R"
 
-#we might not need comments here???
-disqus_shortname: 
+# Contact.  This *must* include the protocol: if it's an email
+# address, it must look like "mailto:lessons@software-carpentry.org",
+# or if it's a URL, "https://gitter.im/username/ProjectName".
+contact: "mailto:lessons@software-carpentry.org"
 
-# Social networking links are used in author-bio sidebar. Update and remove as you like.
+#------------------------------------------------------------
+# Generic settings (should not need to change).
+#------------------------------------------------------------
 
-# Owner/author information
-owner: 		
-  name: NEON / Data Carpentry Hackathon
-  avatar: profilePic.png
-  bio: "Devoted to open data and open source in science and education."
-  email: "neondataskills@neoninc.org"
-#  twitter: "leahawasser"
-  github: "data-lessons"
-#  youtube: " "
-# instagram:      " "
-# For Google Authorship https://plus.google.com/authorship
-# google_plus:    
+# What kind of thing is this ("workshop" or "lesson")?
+kind: "lesson"
 
-# Analytics and webmaster tools stuff goes here
-google_analytics:   
-#google_verify:      UQj93ERU9zgECodaaXgVpkjrFn9UrDMEzVamacSoQ8Y
-# https://ssl.bing.com/webmaster/configure/verify/ownership Option 2 content= goes here
-#bing_verify:        D81F4C18A6CB3018F64D7C827D953DFD
+# Magic to make URLs resolve both locally and on GitHub.
+# See https://help.github.com/articles/repository-metadata-on-github-pages/.
+repository: datacarpentry/R-spatial-raster-vector-lesson
 
-# Links to include in top navigation
-# For external links add external: true
+# Sites.
+amy_site: "https://amy.software-carpentry.org/workshops"
```

Comment:
great @tracykteal  once again... is this the home vs just a carpentry site vs datacarpentry. technically this was DC but now is the carpentries...